ote/src/cljs/ote/views/service_search.cljs

```diff
@@ -241,13 +241,23 @@
 
       [operator-result-chips e! chip-results]]))
 
+(defn- capitalize-operation-area [sentence]
```

Comment:
This function works only for operation areas that is formed using multiple words and numbers such as postal codes.
E.g. "90570 oulu" would work, but "helsinki" would not.